PrototypeArmoury/Config/Items/XComStrategyTuning.ini

```diff
@@ -276,3 +248,59 @@ Cost=(ResourceCosts[0]=(ItemTemplateName="Supplies", Quantity=30), ResourceCosts
 
 [PoweredTemplarArmor_Diff_3 X2ArmorTemplate]
 Cost=(ResourceCosts[0]=(ItemTemplateName="Supplies", Quantity=65), ResourceCosts[1]=(ItemTemplateName="AlienAlloy", Quantity=10), ResourceCosts[2]=(ItemTemplateName="EleriumDust", Quantity=7))
+
+; TODO: make this MCM toggled
+; Turn this option to true to change the cost of hardcoded vanilla items according to the arrItemCostOverrides defined below
+; The overridden prices generally make items cheaper for campaigns where you will need a lot of each item. Not recommended for a vanilla campaign.
+EnableUtilityItemOverrides=false
```

Comment:
Very much 👎 

* CI cannot override this
* You are disabling a system, while naming it by a specific use - what if some mod/bridge uses `arrItemCostOverrides` to change prices only when PA is loaded?

Reply:
In that case, we have two options:
- multiply ItemCostOverrides by the global price multiplier as well
- remove all the ItemCostOverride entries from PA except for a commented example and move them to CI

Reply:
The latter probably makes more sense. The utility items are individual build anyway, their cost re balancing was done for CI